run.sh

```diff
@@ -25,3 +25,5 @@ run "PHP" "php ./php/code.php 40"
 run "R" "Rscript ./r/code.R 40"
 run "Python" "python3 ./py/code.py 40" 
 run "Dart" "./dart/code 40"
+run "C#" "dotnet ./csharp/code/code 40"
+run "F#" "./fsharp/code/code 40"
```

Comment:
Do we have to add "NativeAot" in this? like "C# (NativeAot)" or "F# (NativeAot)"
I'am sure there is the difference between them and other PR does not use native aot #39

Reply:
The other two PRs do use it. It's a good default for console applications. Both AOT and JIT use the same compiler back-end (the differences in optimizations won't show up in such microbenchmark because it does not do anything sensitive to optimizations exclusive to JIT or AOT), and it's more convenient to use AOT as it produces more consistent numbers in the cases like this and obviously cheaper startup cost.